instantsearch/src/main/java/com/algolia/instantsearch/helpers/Searcher.java

```diff
@@ -241,7 +243,8 @@ public void requestCompleted(@Nullable JSONObject content, @Nullable AlgoliaExce
                     if (content == null) {
                         Log.e("Algolia|Searcher", "content is null but error too.");
                     } else {
-                        EventBus.getDefault().post(new ResultEvent(content, query, currentRequestId));
+                        SearchResults results = new SearchResults(content, hierarchicalFacets);
```

Comment:
Might as well inline `results` in the next line

Reply:
Put it inline will make the next line too long.